apps/backend/src/app/alerting/alerting.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Logger, Post } from '@nestjs/common';
+import { ApiOperation } from '@nestjs/swagger';
+import { AlertingService } from './alerting.service';
+import { AlertingInformationDto } from './dto/alertingInformationDto';
+
+@Controller('alerting')
+export class AlertingController {
+  readonly logger = new Logger(AlertingController.name);
+
+  constructor(private readonly alertingService: AlertingService) {}
+
+  @Post()
```

Comment:
Just a thought, but we should really consider having some kind of security/authentication mechanism here. Implemented this way, anyone with access to the API port can trigger E-Mails towards anyone without any authentication, just by posting on this REST endpoint. Since we have no production instance at this point, I think it's not too much of a problem for now, but that will be a must-have in the future.